### PR TITLE
Update README to clarify on fresh installations

### PR DESCRIPTION
**Pull request by zkagin**
Updates the README to help clarify some questions for someone coming to this repository fresh.
 - Fixes formatting/spacing to match Prettier / README best practices.
 - Adds clarifying details for variables.
 - Adds missing steps that were causing errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,11 +47,35 @@ STUDENT_ORG_UNIT=alphanumeric string id for the org unit (retrieve through Admin
 SCHOOL_YEAR_START=YYYY-MM-DD
 ```
 
-3. Generate the `token.pickle` and `credentials.json` files.
+3. Create a folder for logging and caching data.
+
+```
+mkdir data
```
Comment (dchess) [inline README.md:53]: We can add a `.gitkeep` to the data folder to remove the need to manually set this up. Forgot to do that.

File: README.md [hunk 2]
```diff
@@ -15,8 +16,22 @@
...
+- Pipenv is most commonly installed via Homebrew.
+- Docker can be installed directly from the website at docker.com.
+
+```
+brew install pipenv
+```
+
 2. Create .env file with project secrets
 
+Database variables are configured in the format used by [Sqlsorcery](https://github.com/dchess/sqlsorcery).
```
Comment (dchess) [inline README.md:30]: I like this addition. Maybe we can link to that specific section of the sqlsorcery documentation.
https://sqlsorcery.readthedocs.io/en/latest/cookbook/environment.html

**Comment by dchess** [comment]
@susanxiong I invited @zkagin to collab with us on the Google Classroom code. He's made a few edits to the README based on his experience trying to set up the codebase with nothing but the original README. Have a look and approve the merge if it looks good to you.

File: README.md [hunk 3]
```diff
@@ -15,8 +16,22 @@
 git clone https://github.com/kipp-bayarea/google_classroom.git
 ```
 
+2. Install dependencies
+
+- Pipenv is most commonly installed via Homebrew.
+- Docker can be installed directly from the website at docker.com.
+
+```
+brew install pipenv
```
Comment (dchess) [inline README.md:25]: @zkagin We usually leave this broad so it's not system dependent and recommend using a straigh `pip install pipenv` but unless you are contributing to the repo and adding python dependencies, its not necessary to install pipenv as that is handled inside the Docker container.

File: README.md (hunk 1 above)
Comment (zkagin) [reply README.md]: Makes sense, added that instead and removed this step.

File: README.md (hunk 2 above)
Comment (zkagin) [reply README.md]: Switched link to refer to that instead.

File: README.md (hunk 3 above)
Comment (zkagin) [reply README.md]: Makes sense, I generalized this a little bit more. I think it is still necessary right now to get an initial token though?

**Comment by zkagin** [comment]
NOTE: I'm not closely managing the commits to this branch, so we should probably squash them when merging so we don't get a bunch of junk in there :-).

File: README.md (hunk 3 above)
Comment (dchess) [reply README.md]: That is true. I would love to figure out the port config in docker to be able to do tje token generation from within.

**Pull request merged**